shared-helpers/src/formKeys.ts

```diff
@@ -127,6 +141,51 @@ export const sexualOrientation = [
   "notListed",
 ]
 
+const prependRoot = (root: string, subKeys: string[]) => {
+  return subKeys.map((key) => `${root}-${key}`)
+}
+
+interface subCheckboxes {
+  [key: string]: string[]
+}
+
+// Transform an object with keys that may be prepended with a string to an array of only the values with the string
+export const fieldGroupObjectToArray = (
+  formObject: { [key: string]: any },
+  rootKey: string
+): string[] => {
+  const modifiedArray: string[] = []
+  const getValue = (elem: string) => {
+    const formSubKey = elem.substring(elem.indexOf("-") + 1)
+    return formSubKey === formObject[elem] ? formSubKey : `${formSubKey}: ${formObject[elem]}`
+  }
+  Object.keys(formObject)
+    .filter((formValue) => formValue.split("-")[0] === rootKey && formObject[formValue])
+    .forEach((elem) => {
+      if (formObject[elem].isArray) {
+        formObject[elem].forEach(() => {
+          modifiedArray.push(getValue(elem))
+        })
+      } else {
+        modifiedArray.push(getValue(elem))
+      }
+    })
+  return modifiedArray
+}
```

Comment:
Can you write unit tests for this? It looks like the demographics use case doesn't ever hit the check for `formObject[elem].IsArray`. Do we need to support that now?

Reply:
We don't support unit testing of JS in shared-helpers yet on dev, I can pull in my other branch

Reply:
Pulled in changes and wrote tests